Tests/Request/ParamFetcherTest.php

```diff
@@ -553,8 +546,10 @@ public function testKeyPrecedenceOverName()
 
     /**
      * Test an Exception is thrown in strict mode.
+     *
+     * @group legacy
      */
-    public function testConstraintThrowExceptionInStrictMode()
+    public function testConstraintThrowsExceptionWithExpectedMessageInStrictModeIfViolationFormatterIsGiven()
     {
         if (!class_exists('Symfony\Component\HttpFoundation\RequestStack')) {
             $this->markTestSkipped('RequestStack unvailable.');
```

Comment:
there's a typo here (not related to this PR but maybe it's easier to change this here ?)